### DISCUSSION TITLE
[maintenance call] volunteer needed for maintaining centaur-tabs (a spacemacs prerequisite)

### DISCUSSION THREAD

**Discussion by erikmd** [syl20bnr/spacemacs#16279] [Q&A] [answered]
As an emacs *and* spacemacs-theme user, I strongly rely on the [`centaur-tabs`](https://github.com/ema2159/centaur-tabs) mode, which is [a building block](https://github.com/syl20bnr/spacemacs/blob/develop/layers/+emacs/tabs/funcs.el) of spacemacs.

[![centaur-tabs](https://raw.githubusercontent.com/ema2159/centaur-tabs/master/images/underline.png)](https://github.com/ema2159/centaur-tabs#readme)

It happens the `centaur-tabs` project is now looking for maintainers, and given the importance of this mode for spacemacs, we've thought this could be a possible place for posting a call for maintenance.

The question is twofold:

1. Do you have suggestions for other places where this call could be posted? (e.g., by copying [this Discussion URL](https://github.com/syl20bnr/spacemacs/discussions/16279) over there)

2. Would you be interested in applying for maintaining (or co-maintaining) `centaur-tabs`?

For 2., please respond to [this GitHub issue](https://github.com/ema2159/centaur-tabs/issues/228) with your motivation, and a short summary of relevant experience, for becoming a `centaur-tabs` maintainer. The maintainer will be selected from the issue responders by @ema2159.

Thank you for your attention.

**Comment by pataquets** [comment, marked as answer]
[Issue gardening]
@erikmd: After reading the linked issues, I guess this discussion can be closed right away.
I'd like also to thank you as a Spacemacs user for making this happen :heart:.

**Reply by erikmd** [reply]
Hi @pataquets, thanks! Indeed, good point! I just closed the discussion.
